docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -123,7 +123,7 @@ The XFCC header value is a comma (",") separated string. Each substring is an XF
 holds information added by a single proxy. A proxy can append the current client certificate
 information as an XFCC element, to the end of the request's XFCC header after a comma.
 
-Each XFCC element is a semicolon ";" separated string. Each substring is a key-value pair, grouped
+Each XFCC element is a semicolon (";") separated string. Each substring is a key-value pair, grouped
 together by an equals ("=") sign. The keys are case-insensitive, the values are case-sensitive. If
 ",", ";" or "=" appear in a value, the value should be double-quoted. Double-quotes in the value
```

Comment:
I'm not sure this change is worth it, but if you are going to make a change like this I would be consistent throughout. Please also fix DCO, thanks.

/wait

Reply:
Maybe we can wait for more typos for a batch pr. We are doing the translation. Some of the typos confused us.

Reply:
Yes waiting to batch all the typos you find would be appreciated.